CoreFunctionality/test/unit/src/au/gov/asd/tac/constellation/functionality/startup/ConstellationLAFSettingsNGTest.java

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.functionality.startup;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.swing.JFrame;
+import javax.swing.LookAndFeel;
+import javax.swing.UIDefaults;
+import javax.swing.UIManager;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import org.openide.windows.WindowManager;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author OrionsGuardian
+ */
+public class ConstellationLAFSettingsNGTest {
+    
+    private static final Logger LOGGER = Logger.getLogger(ConstellationLAFSettingsNGTest.class.getName());
+    final static PseudoLookAndFeel pseudoLAF = new PseudoLookAndFeel();
+    final static PseudoUIDefaults pseudoUIdefaults = new PseudoUIDefaults();
+    
+    @Test
+    public void runSetLAFTabColors() {
```

Comment:
might be worth splitting this up into separate tests for each class of settings. Firstly it means you can test each one separately (and if one fails it still tests all the subsequent ones), and secondly you can omit the log statements for successfully passing as it will become clear which ones have passed or failed

Reply:
Done. Tests have been split out for different Look and Feel settings. However, I'd like to maintain the log statements. When I run a test manually, I want to see progress information stating what exactly is being tested, rather than just an end result saying "All 12 tests have passed".